mii/client.py

```diff
@@ -156,6 +161,38 @@ def destroy_session(self, session_id):
             client.destroy_session(session_id)
 
 
+class MIINonPersistentClient():
+    def __init__(self, task, deployment_name):
+        self.task = task
+        self.deployment_name = deployment_name
+
+    def query(self, request_dict, **query_kwargs):
+        task_methods = GRPC_METHOD_TABLE[self.task]
+        inference_pipeline = mii.non_persistent_models[self.deployment_name][0]
+
+        if self.task == Tasks.QUESTION_ANSWERING:
+            if 'question' not in request_dict or 'context' not in request_dict:
+                raise Exception(
+                    "Question Answering Task requires 'question' and 'context' keys")
+            args = (request_dict["question"], request_dict["context"])
+            kwargs = query_kwargs
+
+        elif self.task == Tasks.CONVERSATIONAL:
+            conv = create_conversation(request_dict, **query_kwargs)
+            args = (conv, )
+            kwargs = query_kwargs
```

Comment:
```suggestion
            kwargs = {}
```

take a look here:
https://github.com/microsoft/DeepSpeed-MII/blob/dc5ab44dfa48ae9f0a99b356e96c8849c0c78aea/mii/method_table.py#L198

We want `kwargs` to be empty since we're using them above in `create_conversation`